Clarify how to resize images in tables (#333)
obsidianmd/obsidian-docs: en/How to/Format your notes.md

------------ | ------------
[[Format your notes\|Formatting]]	|  [[Use hotkeys\|hotkeys]]	

> If you want to resize images in tables, the pipe must be escaped with a `\` as well:

Reviewer: ```suggestion
If you want to resize images in tables, you need to escape the pipe with a `\`:
```
